OxfordBodleian/Juel-Jensen/BDLaethe30.xml

```diff
@@ -641,11 +641,11 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                                 <seg type="ink">Black, red</seg>
                                 <date notBefore="1800" notAfter="1850">First half of the 19th century</date>
                                 <desc>
-                                    Fine handwriting, sometimes uneven in size and thickness (e.g., <locus target="#23r #24r #29r"/>).
+                                    Fine handwriting, sometimes uneven in size and thickness (e.g. <locus target="#23r #24r #29r"/>).
                                 </desc>
                                 <note>
                                     The graphemes ግ and ዚ in the word <foreign xml:lang="gez">እግዚአብሔር፡</foreign> are occasionally 
-                                    written with ligature (e.g., on <locus target="#14v #16v #31v #49r #82r #113r"/>).
+                                    written with ligature (e.g. on <locus target="#14v #16v #31v #49r #82r #113r"/>).
```

Comment:
"a ligature" ?

Reply:
I trust on your judgement on this. "A ligature" seems appropriate